verifiable-db/src/query/batching/row_chunk/mod.rs

```diff
@@ -0,0 +1,322 @@
+//! This module contains data structures and gadgets employed to build and aggregate
+//! row chunks. A row chunk is a set of rows that have already been aggregated
+//! and whose rows are all proven to be consecutive. The first and last rows in
+//! the chunk are labelled as the `left_boundary_row` and the `right_boundary_row`,
+//! respectively, and are the rows employed to aggregate 2 different chunks.
+
+use mp2_common::{
+    serialization::circuit_data_serialization::SerializableRichField,
+    utils::{FromTargets, HashBuilder, SelectTarget, ToTargets},
+};
+use plonky2::{
+    hash::hash_types::{HashOutTarget, NUM_HASH_OUT_ELTS},
+    iop::target::{BoolTarget, Target},
+    plonk::circuit_builder::CircuitBuilder,
+};
+
+use crate::query::{
+    merkle_path::{MerklePathWithNeighborsTarget, NeighborInfoTarget},
+    universal_circuit::universal_query_gadget::UniversalQueryOutputWires,
+};
+
+/// This module contains gadgets to aggregate 2 different row chunks
+pub(crate) mod aggregate_chunks;
+/// This module contains gadgets to enforce whether 2 rows are consecutive
+pub(crate) mod consecutive_rows;
+
+/// Data structure containing the wires representing the data realted to the node of
```

Comment:
`related`

Reply:
Fixed in commit https://github.com/Lagrange-Labs/mapreduce-plonky2/pull/396/commits/5787dc75a5d6df1a467910e6fc2b524e21943460